common/src/main/java/gov/cms/ab2d/common/model/PGInet.java

```diff
@@ -0,0 +1,63 @@
+package gov.cms.ab2d.common.model;
+
+import java.io.Serializable;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+@SuppressWarnings("serial")
+public class PGInet implements Serializable {
```

Comment:
Why is there a need for this wrapping class?  Why not use InetAddr directly?

You can still have the mapping class to perform the serialization.

Reply:
Hibernate needs the equals and hashcode, any implementation like this will do that https://vladmihalcea.com/postgresql-inet-type-hibernate/